include/assembly.h

```diff
@@ -25,155 +25,6 @@ namespace Assembly
 {
   namespace Scratch
```

Comment:
dove viene usato in interface? io mi riferisco al namespace `Scratch` in interface e altrove usiamo `Scratch` come oggetto della classe `FEValuesCache`